contrib/k8s/examples/sqldb/sqldb-db-only-instance.yaml

```diff
@@ -4,7 +4,7 @@ metadata:
   name: db-only-sqldb-instance
   namespace: default
 spec:
-  clusterServiceClassExternalName: azure-sqldb-db-only
+  clusterServiceClassExternalName: azure-sql-database-only
   clusterServicePlanExternalName: basic
   parameters:
-    parentAlias: dbms-only-sqldb
+    parentAlias: 3e8f59b2-0f73-4a59-9c0b-6277c3e8109e
```

Comment:
How come this switched from a name to a uuid?

Reply:
Suggestion from @krancour . We aren't doing validation on the parentAlias being unique, so he suggested changing it to a uuid. No strong feelings either way for me. 

@krancour ?

Reply:
Within a namespace isn't it unique? Or does alias not refer to the k8s name?

Reply:
It's within the broker, which doesn't do anything w/ k8s related info at the current time, so it's really at the broker level. We could make these names, jsut need to make sure it's unique amongst our examples in case anyone wants to provision multiple examples concurrently.

Reply:
Oops! Sorry I live in k8s land all day so I forget sometimes that there is more to life than just k8s. 😀 No, I understand now. It makes more sense to leave as-is.